oauth2-server/src/main/java/com/clouway/oauth2/authorization/ClientAuthorizationRepository.java

```diff
@@ -2,38 +2,51 @@
 
 import com.clouway.oauth2.DateTime;
 import com.clouway.oauth2.client.Client;
+import com.clouway.oauth2.codechallenge.CodeChallenge;
 import com.google.common.base.Optional;
 
 import java.util.Set;
 
 /**
  * ClientAuthorizationRepository is a repository which is keeping records for the authroziations
  * that are performed for the client applications.
- * 
+ *
  * @author Ivan Stefanov <ivan.stefanov@clouway.com>
  */
 public interface ClientAuthorizationRepository {
 
   /**
    * Finds authorization that is associated with the provided authCode.
    *
-   * @param client the client for which authorization was issued
+   * @param client   the client for which authorization was issued
    * @param authCode the code of the authorization
-   * @param instant the time on which check is performed
+   * @param instant  the time on which check is performed
    * @return the authorization or absent value
    */
   Optional<Authorization> findAuthorization(Client client, String authCode, DateTime instant);
-
+//If we decide to use the new authorize method for all requests we could remove this as it is only used in the example app
```

Comment:
I'm not sure that I understand the goal of these comments. Are they going to be removed before PR is merged or there role is to be in the code  ?

Reply:
This will be removed. Forgot to write it in this one as well.